### DISCUSSION TITLE
Capture those overlays.

### DISCUSSION THREAD

**Discussion by Glurak18** [awSweeney/FFXIV-ACT-Rainbow-Mage-Overlays#23] [Q&A]
Is there a way to capture those overlay with OBS Browsersouce.  I only can capture the basic overlay.

**Comment by awSweeney** [comment]
If I recall there isn't a way to target the overlay specifically. The way I had done it in the past is to do a window/display capture, and set the zones to only capture the part of the screen that the overlay was at.

Doing a search for capture partial or zone of monitor for obs should provide some tutorials.
